cmd/utils/flags.go

```diff
@@ -409,35 +409,15 @@ var (
 	}
 
 	// Gas price oracle settings
-	GpoMinGasPriceFlag = BigFlag{
-		Name:  "gpomin",
-		Usage: "Minimum suggested gas price",
-		Value: big.NewInt(20 * params.Shannon),
-	}
-	GpoMaxGasPriceFlag = BigFlag{
-		Name:  "gpomax",
-		Usage: "Maximum suggested gas price",
-		Value: big.NewInt(500 * params.Shannon),
-	}
-	GpoFullBlockRatioFlag = cli.IntFlag{
-		Name:  "gpofull",
-		Usage: "Full block threshold for gas price calculation (%)",
-		Value: 80,
-	}
-	GpobaseStepDownFlag = cli.IntFlag{
-		Name:  "gpobasedown",
-		Usage: "Suggested gas price base step down ratio (1/1000)",
-		Value: 10,
-	}
-	GpobaseStepUpFlag = cli.IntFlag{
-		Name:  "gpobaseup",
-		Usage: "Suggested gas price base step up ratio (1/1000)",
-		Value: 100,
+	GpoBlocksFlag = cli.IntFlag{
+		Name:  "gpoblocks",
+		Usage: "Number of recent blocks to check for gas prices",
+		Value: 5,
 	}
-	GpobaseCorrectionFactorFlag = cli.IntFlag{
-		Name:  "gpobasecf",
-		Usage: "Suggested gas price base correction factor (%)",
-		Value: 110,
+	GpoPercentileFlag = cli.IntFlag{
+		Name:  "gpopercentile",
+		Usage: "Suggested gas price is the given percentile of a set of recent transaction gas prices",
+		Value: 50,
```

Comment:
This 50th percentile seems a bit arbitrary to me. Perhaps it would be nice to "scan" the mainnet transactions and see what a sane vaue would be for the default parameters.